### ISSUE TITLE
Unable to local invoke python function on windows 10

### ISSUE THREAD

**Issue by RudolfVonKrugstein** [serverless/serverless#3926]
# This is a Bug Report)

## Description

I am trying to "invoke local" my python based function (its python3.6, but I also tried python2.7)

When I do

```
sls invoke local --function myfunction
```

I get:

```
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn python3.6 ENOENT
    at exports._errnoException (util.js:1018:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)
```

The function is working when deployed to aws.
This is my config:

```
service: my-service
custom: ${file(./vars.yml):${opt:stage, self:provider.stage}}

provider:
  name: aws
  runtime: python3.6
  region: eu-central-1
    environment:
    TEMPLATE_BUCKET: "${self:custom.templateBucket}"
    RESULT_BUCKET: "${self:custom.resultBucket}"
functions:
  myfunction:
    handler: handler.run
    events:
        - http:
            path: myfunction/start
            method: post
            cors: true
```

## Additional Data

I am running on windows 10. The serverless version is: 1.17.0

**Comment by dschep** [comment]
Is python available in your `PATH` env var? If not, it needs to be (check that box in the python installer).

Or, if you're using a virtualenv, this should be fixed by #4286, which was just released in sls 1.23

**Comment by pmuens** [comment]
Closing since we've fixed lots of `invoke local` fuctionalities in the past. Feel free to re-open if this is still a problem.
